package.json

```diff
@@ -27,7 +27,8 @@
   },
   "pnpm": {
     "overrides": {
-      "vike-node": "link:./packages/vike-node/"
+      "vike-node": "link:./packages/vike-node/",
+      "esbuild": "^0.24.0"
```

Comment:
I guess we can remove this line?

Reply:
There are a lot of discrepancies between all esbuild versions installed without it.
Usually it doesn't cause issues, but when it does, it always takes times to debug (it happened at least 3 times on Bati). So I just usually add this when I know this can happen.

```sh
# pnpm -r why

/home/magne/workspace/vike-node

devDependencies:
@brillout/test-e2e 0.5.33
└── esbuild 0.16.17

/home/magne/workspace/vike-node/examples/express-react

dependencies:
@vitejs/plugin-react 4.3.1
└─┬ vite 5.4.8 peer
  └── esbuild 0.21.5
vike 0.4.198
├── esbuild 0.23.1
└─┬ vite 5.4.8 peer
  └── esbuild 0.21.5
vike-node link:../../packages/vike-node
└── esbuild 0.24.0
vike-react 0.5.7
└─┬ vike 0.4.198 peer
  ├── esbuild 0.23.1
  └─┬ vite 5.4.8 peer
    └── esbuild 0.21.5
vite 5.4.8
└── esbuild 0.21.5

/home/magne/workspace/vike-node/examples/express-react-vercel

dependencies:
@vitejs/plugin-react 4.3.1
└─┬ vite 5.4.8 peer
  └── esbuild 0.21.5
vike 0.4.198
├── esbuild 0.23.1
└─┬ vite 5.4.8 peer
  └── esbuild 0.21.5
vike-node link:../../packages/vike-node
└── esbuild 0.24.0
vike-react 0.5.7
└─┬ vike 0.4.198 peer
  ├── esbuild 0.23.1
  └─┬ vite 5.4.8 peer
    └── esbuild 0.21.5
vite 5.4.8
└── esbuild 0.21.5

/home/magne/workspace/vike-node/examples/hono-react-cf-pages

dependencies:
@vitejs/plugin-react 4.3.1
└─┬ vite 5.4.8 peer
  └── esbuild 0.21.5
vike 0.4.198
├── esbuild 0.23.1
└─┬ vite 5.4.8 peer
  └── esbuild 0.21.5
vike-node link:../../packages/vike-node
└── esbuild 0.24.0
vike-react 0.5.7
└─┬ vike 0.4.198 peer
  ├── esbuild 0.23.1
  └─┬ vite 5.4.8 peer
    └── esbuild 0.21.5
vite 5.4.8
└── esbuild 0.21.5

devDependencies:
wrangler 3.71.0
├─┬ @esbuild-plugins/node-globals-polyfill 0.2.3
│ └── esbuild 0.17.19 peer
├─┬ @esbuild-plugins/node-modules-polyfill 0.2.2
│ └── esbuild 0.17.19 peer
└── esbuild 0.17.19

/home/magne/workspace/vike-node/examples/hono-react-vercel-edge

dependencies:
@vitejs/plugin-react 4.3.1
└─┬ vite 5.4.8 peer
  └── esbuild 0.21.5
vike 0.4.198
├── esbuild 0.23.1
└─┬ vite 5.4.8 peer
  └── esbuild 0.21.5
vike-node link:../../packages/vike-node
└── esbuild 0.24.0
vike-react 0.5.7
└─┬ vike 0.4.198 peer
  ├── esbuild 0.23.1
  └─┬ vite 5.4.8 peer
    └── esbuild 0.21.5
vite 5.4.8
└── esbuild 0.21.5

vike-node@0.1.16 /home/magne/workspace/vike-node/packages/vike-node

dependencies:
esbuild 0.24.0

devDependencies:
tsup 8.3.0
├─┬ bundle-require 5.0.0
│ └── esbuild 0.23.1 peer
└── esbuild 0.23.1
universal-middleware 0.3.8
├── esbuild 0.24.0 peer
└─┬ vite 5.4.8 peer
  └── esbuild 0.21.5
vike 0.4.198
├── esbuild 0.23.1
└─┬ vite 5.4.8 peer
  └── esbuild 0.21.5
vite 5.4.8
└── esbuild 0.21.5

/home/magne/workspace/vike-node/test/vike-node

dependencies:
@vitejs/plugin-react 4.3.1
└─┬ vite 5.4.8 peer
  └── esbuild 0.21.5
vike 0.4.198
├── esbuild 0.23.1
└─┬ vite 5.4.8 peer
  └── esbuild 0.21.5
vike-node link:../../packages/vike-node
└── esbuild 0.24.0
vite 5.4.8
└── esbuild 0.21.5
```

Reply:
I see. (Vike's esbuild usage is minimalistic so I didn't experience that.)

A little concern I have about this is that it may lead to wrong esbuild semver ranges in `package.json` files. But I guess it isn't that important. I'm marking this conversation as resolved.

(FYI [Vike has a wide range](https://github.com/vikejs/vike/blob/05e5bd53e09c8a4e819e4bd8021147cd52249100/vike/package.json#L132) because Vite is fairly conservative about which esbuild it uses, so I'm trying to avoid a double esbuild installation. But if you feel more comfortable using the latest esbuild version then we can do that.)